text/0000-light-dom.md

```diff
@@ -236,7 +236,40 @@ To support the cases where a shadow DOM element composes a light element, light
 
 It is important to notice that the order in which light DOM components are rendered impacts the order in which stylesheets are injected into the root node and directly influences CSS rule specificity.
 
-**Note:** Different approaches to layer style scoping on top has been discussed while designing Light DOM, like introducing a new file extension for automatic style scoping (`.scoped.css`) or using a `<style scoped>` element in the template. This can be addressed in a future RFC.
+##### `:host` and `:host-context()`
+
+Also worth noting is that `:host` and `host-context()` are inserted as-is. Therefore, a light DOM component can style its
```

Comment:
I would call out that this is behaving according to the specification. 
The `:host` and `:host-context` also only match if the root node is a shadow root. Those selectors are ignored when injected at the document level.

Reply:
Good point, I'll say that.